libs/net/dal/Migrations/1.0.14/Up/PostUp/02-Source.sql

```diff
@@ -0,0 +1,162 @@
+DO $$
+BEGIN
+
+UPDATE public."source" SET
```

Comment:
One downside to this implementation is if the source is missing it won't be added/updated.